Automattic/jetpack
# Admin Menu: Adding tests to test Jetpack admin menu 'current' menu item class and menu order

Fixes https://github.com/Automattic/vulcan/issues/482

## Proposed changes:

* This PR adds a test that checks if the 'current' class gets applied to one of the Jetpack sub-menu items that we are adding the 'current' class to (logic added in this PR: https://github.com/Automattic/jetpack/pull/39095)
* It also adds a test to confirm the order of 9 Jetpack menu items (available via the Jetpack plugin, plus available packages): My Jetpack, Stats, VideoPress, VaultPress Backup, Activity Log, Search, WordAds, Settings and Dashboard (confirming they are always in that order, even if not next to each other).
* I've left Jetpack Manage as it only valid for a subset of sites (also only for users with more than 2 connected sites), and because of this: pf36In-WA-p2

### Other information:


## Jetpack product discussion

https://github.com/Automattic/vulcan/issues/482

## Does this pull request change what data or activity we track or use?

No.

## Testing instructions:

* To test locally, navigate to `projects/plugins/jetpack` and run `npm run test-adminpage`. All the tests should pass. This includes the test for the 'current' menu item class addition.
* For the menu order test, run `jetpack docker phpunit -- --filter=test_jetpack_admin_menu_order` - the test should pass
